GameDataParser/Parsers/SkillParser.cs

```diff
@@ -189,30 +190,85 @@ private static void ParseConditionSkill(XmlNode parentNode, List<SkillCondition>
             uint delay = uint.Parse(conditionNode.Attributes["delay"].Value ?? "0");
             int removeDelay = int.Parse(conditionNode.Attributes["removeDelay"]?.Value ?? "0");
             bool useDirection = int.Parse(conditionNode.Attributes["useDirection"]?.Value ?? "0") == 1;
+            bool randomCast = int.Parse(conditionNode.Attributes["randomCast"]?.Value ?? "0") == 1;
+            int[] linkSkillID = conditionNode.Attributes["linkSkillID"]?.Value?.SplitAndParseToInt(',')?.ToArray() ?? new int[0];
+            int overlapCount = int.Parse(conditionNode.Attributes["overlapCount"]?.Value ?? "0");
+            bool nonTargetActive = int.Parse(conditionNode.Attributes["nonTargetActive"]?.Value ?? "0") == 1;
+            bool onlySensingActive = int.Parse(conditionNode.Attributes["onlySensingActive"]?.Value ?? "0") == 1;
+            bool dependOnCasterState = int.Parse(conditionNode.Attributes["dependOnCasterState"]?.Value ?? "0") == 1;
+            bool activeByIntervalTick = int.Parse(conditionNode.Attributes["activeByIntervalTick"]?.Value ?? "0") == 1;
+            bool dependOnDamageCount = int.Parse(conditionNode.Attributes["dependOnDamageCount"]?.Value ?? "0") == 1;
 
             skillConditions.Add(new(conditionSkillId, conditionSkillLevel, splash, target, owner, fireCount, interval, immediateActive)
             {
                 Delay = delay,
                 RemoveDelay = removeDelay,
                 UseDirection = useDirection,
+                RandomCast = randomCast,
+                LinkSkillId = linkSkillID,
+                OverlapCount = overlapCount,
+                NonTargetActive = nonTargetActive,
+                OnlySensingActive = onlySensingActive,
+                DependOnCasterState = dependOnCasterState,
+                ActiveByIntervalTick = activeByIntervalTick,
+                DependOnDamageCount = dependOnDamageCount,
                 BeginCondition = ParseBeginCondition(conditionNode)
             });
-            ;
         }
     }
 
+    private static Dictionary<string, bool> statIgnoreList = new()
+    {
+        ["hp"] = true,
+        ["func"] = true
+    };
+
+    private static Dictionary<string, bool> statIgnoreList2 = new()
+    {
+        ["hp"] = true,
+        ["sp"] = true,
+        ["ep"] = true, // 10500153
+    };
+
     private static SkillBeginCondition ParseBeginCondition(XmlNode parent)
     {
         SkillBeginCondition beginCondition = null;
 
+        int count = 0;
+
         foreach (XmlNode beginNode in parent.SelectNodes("beginCondition"))
         {
+            foreach (XmlNode compareStat in beginNode.SelectNodes("stat"))
+            {
+                foreach (XmlAttribute attribute in compareStat.Attributes)
+                {
+                    if (!statIgnoreList2.ContainsKey(attribute.Name))
+                    {
+                        count += 0;
```

Comment:
What is this line doing?

Reply:
Oh woops, I was using that to test if a case ever happened with breakpoints. Forgot to remove a few of these. I can probably just leave a comment in its place saying that the case never appears in the xmls